### PR TITLE
Add `BigInteger#{int,long}ValueExact`

### PR DESCRIPTION
**Pull request by armanbilge**
The implementation is based on what I found in GWT.
https://github.com/scala-js/scala-js/blob/fce07f56318b5b6e70f695492fc88fec04995428/javalib/src/main/scala/java/math/BigInteger.scala#L1-L5

### CODE REVIEW

File: javalib/src/main/scala/java/math/BigInteger.scala [hunk 1]
```diff
@@ -501,6 +501,12 @@ class BigInteger extends Number with Comparable[BigInteger] {
 
   override def intValue(): Int = sign * digits(0)
 
+  def intValueExact(): Int =
```
Comment (sjrd) [inline javalib/src/main/scala/java/math/BigInteger.scala:504]:
```suggestion
  def intValueExact(): Int = {
```

File: javalib/src/main/scala/java/math/BigInteger.scala [hunk 2]
```diff
@@ -511,6 +517,12 @@ class BigInteger extends Number with Comparable[BigInteger] {
     sign * value
   }
 
+  def longValueExact(): Long =
```
Comment (sjrd) [inline javalib/src/main/scala/java/math/BigInteger.scala:520]:
```suggestion
  def longValueExact(): Long = {
```

**Review by sjrd — CHANGES_REQUESTED** [review]
Could you add tests for boundary conditions: `MinValue`, `MinValue - 1`, `MaxValue` and `MaxValue + 1` (for both `Int` and `Long`)?

**Review by sjrd — APPROVED** [review]
